tests/test_BootstrapReport.py

```diff
@@ -4,7 +4,7 @@
 from scipy.stats import norm, uniform
 from scipy.optimize import minimize
 import pytest
-from BootstrapReport.BootstrapReport import ObjectOfInterest
+from BootstrapReport.report import ObjectOfInterest
```

Comment:
Could something like this nomenclature work better? (Seems at least [somewhat conventional](https://stackoverflow.com/questions/37217786/is-there-a-set-name-i-need-to-give-my-py-file-containing-the-main-function), which is a plus.)

If you like it maybe we can rename `report.py`-->`main.py` and update throughout? 

```suggestion
from BootstrapReport.main import ObjectOfInterest
```